Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py

```diff
@@ -131,6 +131,51 @@ def get_external_ip_address_range_request(
 
         return response
 
+    def get_attack_surface_rule_request(
+        self, attack_surface_rule_id: str, enabled_status: str, priority: str, category: str
+    ) -> Dict[str, Any]:
+        """Get Attack Surface Rule details for an attack surface rule id using the '/get_attack_surface_rules/' endpoint.
+
+        Args:
+            attack_surface_rule_id (str): Coma separated attack surface rule ids.
+
+        Returns:
+            dict: dict containing information about Attack surface rule.
+        """
+        filters = []
```

Comment:
We have added this conditional logic in the _command function rather than the _request method for all other cases.  We should do the same here.

Reply:
Updated in this commit: 0ad1560d46701be1f51498172d3236454085f87f